ChangeLog.md

```diff
@@ -17,6 +17,8 @@ See docs/process.md for how version tagging works.
 
 Current Trunk
 -------------
+- Add warning when using python literal as ``EM_CONFIG`` value.  Hopefully
```

Comment:
please add something like `(that is, when `EM_CONFIG` is not the name of a config file, but actually contains the contents of such a file inline)`